public/pages/DetectorResults/utils/tableUtils.tsx

```diff
@@ -74,6 +74,30 @@ export const staticColumn = [
     truncateText: false,
     dataType: 'number',
   },
+  {
+    field: 'actions',
+    name: (
+      <EuiText size="xs" style={columnStyle}>
+        <b>Actions</b>{' '}
+        <EuiIconTip
+          content="This will create an index pattern with the indices used to create this detector (if it doesn't exist) and open the anomaly logs in Discover."
+          position="top"
+          type="iInCircle"
+        />
+      </EuiText>
+    ),    align: 'left',
+    truncateText: true,
+    actions: [
+      {
+        type: 'icon',
+        name: 'View in Discover',
+        description: 'View in Discover',
+        icon: 'editorLink',
+        onClick: () => {},
+        'data-test-subj': 'discoverIcon',
```

Comment:
apologize i dont have the insight on if this ends up getting info prepended to this test subj but the suggestion would be to make this specific to this page. to avoid potential conflicts. if tmrw the navigation bar adds a button with the name `discoverIcon` then cypress might find the wrong test subj

Reply:
I agree we should make test subj unique enough, but we should be fine here :)